### PR TITLE
Fix grammar in level 3 description

### PR DESCRIPTION
**Pull request by babolivier**

### CODE REVIEW

**Review by couci — APPROVED** [review]
thanks for your contribution

**Review by josehz5578 — APPROVED** [review]

**Comment by janiaz708** [comment]
hi

File: code-of-conduct-enforcement/consequence-ladder.md
Comment (Md0135) [inline code-of-conduct-enforcement/consequence-ladder.md:1]: Türkiye

**Review by Md0135 — COMMENTED** [review]
Türkiye

**Review by Md0135 — COMMENTED** [review]
Türkiye

**Comment by janiaz708** [comment]
hi

**Review by PAOLASSS999 — COMMENTED** [review]
code-of-conduct-enforcement/consequence-ladder.md

**Comment by Setarland** [comment]
abb1a6846e71f371d044b09ccd50e9dd5914c856

**Review by Setarland — APPROVED** [review]

**Comment by Mikypearl** [comment]
Hello

**Review by Ebaoone — APPROVED** [review]

**Review by Ebaoone — APPROVED** [review]

File: code-of-conduct-enforcement/consequence-ladder.md
Comment (Ebaoone) [inline code-of-conduct-enforcement/consequence-ladder.md:1]: f887bc3932bb5dc72ee0bd0bda3985266bdb5be4

**Review by Tihana08 — APPROVED** [review]

**Review by Daveydave101 — COMMENTED** [review]
T

**Review by Daveydave101 — APPROVED** [review]
[]()

**Review by Daveydave101 — CHANGES_REQUESTED** [review]
[]()

**Review by Daveydave101 — COMMENTED** [review]
[]()

File: code-of-conduct-enforcement/consequence-ladder.md
Comment (Daveydave101) [reply code-of-conduct-enforcement/consequence-ladder.md:1]: 2b5367b1525f67709fb3485c873847db646a3699

**Review by Daveydave101 — APPROVED** [review]

**Review by Daveydave101 — APPROVED** [review]
- [Copper Boom ]

**Review by Dustin4444 — APPROVED** [review]

**Pull request merged**